utils/setools/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=setools
+PKG_VERSION:=4.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/setools/releases/download/4.3.0
+PKG_HASH:=315df3ae0eb29b399123c5e3330480c5d1c0da038671c9fd62a439c49a6f9105
+PKG_BUILD_DIR:=$(BUILD_DIR)/setools
+
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=Cython # Cython>=0.27
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+PKG_LICENSE:=GPL-2.0-or-later LGPL-2.1-or-later
+PKG_LICENSE_FILES:=COPYING COPYING.GPL COPYING.LGPL
+PKG_CPE_ID:=cpe:/a:selinuxproject:setools
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/setools
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+python3 +libselinux +libsepol
```

Comment:
I [previously noted](https://github.com/openwrt/packages/pull/10664#issuecomment-671830290) that setools has runtime dependencies on setuptools and networkx... They are mentioned in the [Installation](https://github.com/SELinuxProject/setools#installation) section of the readme.

Looking through their code, I believe you will get import errors if you try to run any of the command-line tools, like `seinfo` or `sediff`, without these dependencies installed.

Regarding setuptools, (again from looking at their code) it looks like they are not using the full setuptools package, only pkg_resources, so including `+python3-pkg-resources` should fulfill that dependency.

Regarding [networkx](https://pypi.org/project/networkx/), there is no Python library package for this package yet. It looks like a relatively simple library package; you should be able to copy an existing Makefile like the one for [python-attrs](https://github.com/openwrt/packages/blob/6333ef24c7fc550b3a2c459ac5eb8e9ea0fc0e20/lang/python/python-attrs/Makefile) and replace the appropriate values.

On networkx's dependencies, I believe python3-light is enough (though I have not verified this). It also has a [dependency on decorator](https://github.com/networkx/networkx/blob/networkx-2.5/setup.py#L127), which is already available in the feed as [python3-decorator](https://github.com/openwrt/packages/blob/6333ef24c7fc550b3a2c459ac5eb8e9ea0fc0e20/lang/python/python-decorator/Makefile).

Reply:
I hope to do this soon.

Reply:
Is that strictly required for this PR already? Else I'd merge this to allow smaller PRs in the future

Reply:
I suppose setools can be marked with `@BROKEN`, but are there rules in this repo on intentionally committing broken packages?

I don't have a strong opinion in this case (personally I wouldn't want to knowingly commit broken code to a repo with such a large audience), I just wanted to point out that there are unmet dependencies.

Reply:
No that's totally reasonable, let's wait for a port of `networkx` :)

Reply:
I pushed an update with the networkx package.